### PR TITLE
feat(lint): Added plugins and rules for sorting imports

### PR DESCRIPTION
**Pull request by lofgrenfredrik**
Added the packages: eslint-plugin-simple-import-sort and eslint-plugin-import.
Added some default import rules.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -57,11 +66,26 @@ module.exports = {
         depth: 3,
       },
     ],
+    'simple-import-sort/sort': 'error',
```
Comment (michaellopez) [inline index.js:69]: @lofgrenfredrik Could you explain a little bit what these rules do?

File: index.js [hunk 2]
```diff
@@ -57,11 +66,26 @@ module.exports = {
...
+    'import/newline-after-import': 'error',
+    'import/no-duplicates': 'error',
   },
   settings: {
     react: {
       version: 'detect',
     },
     linkComponents: [{ name: 'Link', linkAttribute: 'to' }],
   },
+  overrides: [
```
Comment (michaellopez) [inline index.js:81]: @lofgrenfredrik And also explain these

**Review by michaellopez — CHANGES_REQUESTED** [review]
Looks good, just want some clarification on what the new plugins and rules do and so. Short descriptions and some links is enough in the review comments here.

**Comment by lofgrenfredrik** [comment]
The default riles are based on the [example configuration](https://www.npmjs.com/package/eslint-plugin-simple-import-sort#example-configuration) from `eslint-plugin-simple-import-sort` npm site.
The default rules are:
- `simple-import-sort/sort` is turned on by default.
- The standard `sort-imports` rule is turned off, in case you extend a config that includes it.
- `import/first` makes sure all imports are at the top of the file. (autofixable)
- `import/newline-after-import` makes sure there’s a newline after the imports. (autofixable)
- `import/no-duplicates merges` import statements of the same file. (autofixable, mostly)

For Node.js code, ie: the `lambdas` in our `/api` forlder
- `simple-import-sort/sort` is turned off and replaced with `import/order` for sorting of `require` calls.

File: index.js (hunk 1 above)
Comment (lofgrenfredrik) [reply index.js:69]: @michaellopez Added a separate comment including short list and explanation of the activated rules. 👍

File: index.js (hunk 2 above)
Comment (lofgrenfredrik) [reply index.js:81]: Included in a separate comment. 👍

**Review by michaellopez — APPROVED** [review]

**Pull request merged**